types/index.d.ts

```diff
@@ -3,13 +3,19 @@ export = hybrids;
 export as namespace hybrids;
 
 declare namespace hybrids {
+  interface InvalidateOptions {
+    clearValue?: boolean;
```

Comment:
For now I would not expose `clearValue` and `deleteEntry`. The `force` option will do the same and more than `clearValue`, and `deleteEntry` is useless in that context (to call as invalidation of property value).